scripts/rnnlm/prepare_rnnlm_dir.sh

```diff
@@ -157,13 +157,16 @@ if [ $stage -le 7 ]; then
     num_splits=$(cat $dir/text/info/num_splits)
     text_files=$(for n in $(seq $num_splits); do echo -n $dir/text/$n.txt ''; done)
     vocab_size=$(tail -n 1 $dir/config/words.txt | awk '{print $NF + 1}')
+    bos_symbol=`grep "<s>" $dir/config/words.txt | awk '{print $2}'`
```

Comment:
see if you can avoid adding lines like this, it's not very robust (what if a word contains this?).

See $dir/special_symbol_opts.txt.  You may be able to get the options from there.